src/metabase/lib/breakout.cljc

```diff
@@ -69,12 +69,21 @@
 
   ([query        :- ::lib.schema/query
     stage-number :- :int]
-   (let [existing-breakouts (breakouts query stage-number)
-         existing-breakout? (fn [x]
-                              (some (fn [existing-breakout]
-                                      (lib.equality/= (lib.ref/ref x) existing-breakout))
-                                    existing-breakouts))
-         columns            (let [stage (lib.util/query-stage query stage-number)]
-                              (lib.metadata.calculation/visible-columns query stage-number stage))]
+   (let [indexed-breakouts (map-indexed vector (breakouts query stage-number))
+
+         breakout-pos
+         (fn [x]
```

Comment:
nit: this indentation is pretty hard to follow imo.

Reply:
Changed.